frontend/fingertips-frontend/eslint.config.mjs

```diff
@@ -10,11 +10,10 @@ const compat = new FlatCompat({
   recommendedConfig: js.configs.recommended,
   allConfig: js.configs.all,
 });
-export default [
+export default js.configs = [
```

Comment:
just out of interest why was has this been named and why was the rule removed?

Reply:
i was getting an eslint warning for unnamed default export 

the rule was removed as it was for a cypress file which has been removed - so no longer need the rule